cpp/include/cudf/io/functions.hpp

```diff
@@ -515,9 +519,19 @@ struct write_parquet_args {
  * @param args Settings for controlling writing behavior
  * @param mr Optional resource to use for device memory allocation
  */
-void write_parquet(write_parquet_args const& args, rmm::mr::device_memory_resource* mr =
-                                               rmm::mr::get_default_resource());
+std::unique_ptr<std::vector<uint8_t>> write_parquet(
```

Comment:
Could you add to the doxygen that this now returns a unique_ptr to a metadata blob?

Reply:
Done